docs/current_docs/partials/_install-cli.mdx

```diff
@@ -146,3 +148,17 @@ where.exe dagger
 
 </TabItem>
 </Tabs>
+
+### Development version
+
+:::warning
+Development versions should be considered unstable and may produce unexpected results.
```

Comment:
_should be considered unfinished._

We don't expect them to produce unexpected results, since the tests ensure they work as intended.

Reply:
Done